src/app/pages/send-crypto-asset/components/memo-field.tsx

```diff
@@ -1,6 +1,8 @@
 import { TextInputField } from './text-input-field';
 
-interface MemoFieldProps {}
+interface MemoFieldProps {
+  lastChild?: boolean;
+}
```

Comment:
Figured we might need a prop like this. Were we not in css-in-js land `:last-child` would suffice. Was just reading https://webkit.org/blog/13096/css-has-pseudo-class